Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml

```diff
@@ -340,51 +343,107 @@ script:
-    description: List alerts triggered in the last week
+    description: List alerts triggered in the last week.
     arguments:
     - name: portfolio_id
-      description: Portfolio ID. Can be retrieved using `!securityscorecard-portfolios-list`
+      description: Portfolio ID. Can be retrieved using `!securityscorecard-portfolios-list`.
     outputs:
     - contextPath: SecurityScorecard.Alerts.Alert.id
-      description: Alert ID
+      description: Alert ID.
       type: String
     - contextPath: SecurityScorecard.Alerts.Alert.email
       description: Alert email recipient.
       type: String
     - contextPath: SecurityScorecard.Alerts.Alert.change_type
-      description: Alert change type configured (score or threshold)
+      description: Alert change type configured (score or threshold).
       type: String
     - contextPath: SecurityScorecard.Alerts.Alert.domain
-      description: Alert domain
+      description: Alert domain.
       type: String
     - contextPath: SecurityScorecard.Alerts.Alert.company_name
-      description: Alert company name
+      description: Alert company name.
       type: String
     - contextPath: SecurityScorecard.Alerts.Alert.Portfolio.id
-      description: Alert Portfolio ID
+      description: Alert Portfolio ID.
       type: array
     - contextPath: SecurityScorecard.Alerts.Alert.my_scorecard
       description: Whether the alert was triggered on private scorecard. This depends on whether 'my_scorecard' was added to the optional argument 'target' when creating alerts using the 'securityscorecard-alert-score-threshold-create' and 'securityscorecard-alert-grade-change-create' commands.
       type: Boolean
     - contextPath: SecurityScorecard.Alerts.Alert.created_at
-      description: Timestamp of when the alert was triggered
+      description: Timestamp of when the alert was triggered.
       type: Date
   - name: securityscorecard-company-services-get
-    description: Retrieve the service providers of a domain
+    description: Retrieve the service providers of a domain.
     arguments:
     - name: domain
-      description: Company domain
+      description: Company domain.
       required: true
     outputs:
     - contextPath: SecurityScorecard.Service.vendor_domain
-      description: Vendor domain, e.g. Google, Amazon
+      description: Vendor domain, e.g. Google, Amazon.
       type: String
     - contextPath: SecurityScorecard.Service.client_domain
-      description: Client domain. This value is identical to the input of the domain argument
+      description: Client domain. This value is identical to the input of the domain argument.
       type: String
     - contextPath: SecurityScorecard.Service.categories
-      description: Vendor service provider, e.g. mail_provider, nameserver_provider
+      description: Vendor service provider, e.g. mail_provider, nameserver_provider.
       type: array
+  - name: securityscorecard-company-events-get
```

Comment:
```suggestion
  - name: securityscorecard-get-company-events
 ```

Reply:
Well, I see all existing commands in this package already have the pattern in question.  I rather keep consistency.  This package was built by Kobbi in the past